cmd/cored/main.go

```diff
@@ -116,6 +117,17 @@ func init() {
 }
 
 func main() {
+	v := flag.Bool("version", false, "prints cored version")
+	flag.Parse()
+
+	if *v {
+		fmt.Printf("cored version: %s production=%t\n", config.Version, config.Production)
```

Comment:
actually, can we put the `production=%t` on a separate line? it would also be nice to put the buildCommit and buildDate as well, since they're available.

I want to use this format:

```
cored 1.1
production=true
build-commit=abc123
build-date=2017-02-13T21:28:52Z
```
or

```
cored 1.1+changes
production=false
build-commit=?
build-date=?
```

(I'll do the same in signerd.)